includes/Modules/Analytics.php

```diff
@@ -122,10 +122,21 @@ function() {
 		);
 
 		$print_amp_gtag = function() {
+			// This hook is only available in AMP plugin version 1.3+, so if it
+			// has already completed, do nothing.
+			if (
+				! doing_action( 'amp_print_analytics' ) &&
+				did_action( 'amp_print_analytics' )
+			) {
+				return;
+			}
+
 			$this->print_amp_gtag();
 		};
-		add_action( 'wp_footer', $print_amp_gtag ); // For AMP Native and Transitional.
-		add_action( 'amp_post_template_footer', $print_amp_gtag ); // For AMP Reader.
+		// Print analytics when using the AMP Plugin (https://amp-wp.org/).
+		add_action( 'amp_print_analytics', $print_amp_gtag );
+		add_action( 'wp_footer', $print_amp_gtag, 20 ); // For AMP Native and Transitional.
```

Comment:
This could be changed to `For AMP Standard and Transitional, AMP plugin version <1.3.`